test/org/demiurg906/kotlin/plugin/runners/BaseTestRunner.kt

```diff
@@ -28,6 +33,13 @@ abstract class BaseTestRunner : AbstractKotlinCompilerTest() {
 
 fun TestConfigurationBuilder.commonFirWithPluginFrontendConfiguration() {
     baseFirDiagnosticTestConfiguration()
+    commonConfigurationForTest(
```

Comment:
1. `baseFirDiagnosticTestConfiguration` partially duplicates `commonConfigurationForTest`
2. `commonConfigurationForTest` should not be called for diagnostic tests

Reply:
For 1, `baseFirDiagnosticTestConfiguration` is used on diagnostic too, so the duplicated logic can't be deleted based on 2.

For 2, I have moved it to the configuration of `AbstractBoxTest`.